### ISSUE TITLE
How to alias selected column

### ISSUE THREAD

**Issue by seraphjiang** [opendistro-for-elasticsearch/sql#649]
I have a sql like below and try to run in 7.4
```
SELECT t1.Id as ID 
FROM table1 t1
  LEFT JOIN table2 t2
ON
  t1.id = t2.id
```

below is what i got with sql
Actual result
```
t1.Id
---
1
2
3
```

this is what expected.

Expected result
```
ID
--
1
2
3
```

Does SQL support alias selected column?

**Comment by dai-chen** [comment]
Just tried a similar query in DevTools. The response in default format looks right. This may be an issue in SQL Workbench. @chloe-zh Do we support column alias in SQL Workbench. I didn't find open issue related.

```
POST _opendistro/_sql
{  
  "query": """
      select a1.city AS c from accounts a1 LEFT JOIN accounts a2 ON a1.age = a2.age
      """
}
{
  "schema": [{
    "name": "a1.city",
    "alias": "c",
    "type": "text"
  }],
  "total": 200,
  "datarows": [
    ["Rote"],
    ["Beyerville"],
    ["Mahtowa"],
    ["Nanafalia"],
```

**Comment by chloe-zh** [comment]
Hi @seraphjiang , what version of workbench are you using? Please checkout v1.9.0.0 or later since the issue (https://github.com/opendistro-for-elasticsearch/sql-workbench/issues/45) was fixed right before v1.9 was released.

**Comment by seraphjiang** [comment]
we are using 1.4.0.0, elasticsearch 7.4.2, we plan to upgrade to 7.7 soon.
